source/reference/access-mdb-shell-help.txt

```diff
@@ -0,0 +1,241 @@
+                               collection information, i.e. collection
+                               name and options, for the current
+                               database.
+       runCommand              Runs an arbitrary command on the
+                               database. 
+       adminCommand            Runs an arbitrary command against the
+                               admin database. 
+       aggregate               Runs a specified admin/diagnostic
+                               pipeline which does not require an
+                               underlying collection.
+       getSiblingDB            Returns another database without
+                               modifying the db variable in the shell
+                               environment. 
+       getCollection           Removes the current database, deleting
+                               the associated data files.
+       dropDatabase            Removes the current database, deleting
+                               the associated data files.
+
+- To see help for a specific database method in ``mongosh``, type the
+  ``db.<method name>`` without the parenthesis (``()``), followed by
+  ``.help`` or ``.help()``. The following example returns help for the
+  :method:`db.adminCommand()` method:
+
+  .. include:: /includes/admonitions/help-interchangeable.rst
+
+  .. code-block:: javascript
+
+     db.adminCommand.help()
+
+  The output resembles the following:
+
+  .. code-block:: none
+
+       db.adminCommand({ serverStatus: 1 }):
+
+       Runs an arbitrary command against the admin database.
+
+       For more information on usage: https://docs.mongodb.com/manual/reference/method/db.adminCommand
+
+- To see additional usage details for a database method in ``mongosh``,
+  type the ``db.<method name>`` without the parenthesis (``()``). The
+  following example returns details about the
+  :method:`db.adminCommand()` method:
+
+  .. code-block:: javascript
+
+     db.adminCommand
+
+  The output resembles the following:
+
+  .. code-block:: javascript
+     :copyable: false
+
+     [Function] {
+       serverVersions: [ '3.4.0', '4.4.0' ],
+       returnsPromise: true,
+       topologies: [ 0, 2, 1 ],
+       returnType: { type: 'unknown', attributes: {} },
+       platforms: [ 0, 1, 2 ],
+       help: [Function] Help
+     }
+
+
+.. _mdb-shell-help-collection:
+
+Collection Help
+---------------
+
+In ``mongosh``:
+
+- To see the list of collections in the current database, use the
+  ``show collections`` command:
+
+  .. code-block:: javascript
+
+     show collections
+  
+- To see the help for methods available on the collection objects
+  (e.g. ``db.<collection>``), use the ``db.<collection>.help()``
+  method:
+
+  .. include:: /includes/admonitions/help-interchangeable.rst
+
+  .. code-block:: javascript
+
+     db.collection.help()
+
+  ``<collection>`` can be the name of a collection that exists,
+  although you may specify a collection that doesn't exist.
```

Comment:
I think we can simplify this a bit:

You can specify the name of either an existing or non-existing collection.

Reply:
updated